src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/convenience/key_vault_client.py

```diff
@@ -0,0 +1,181 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long,too-many-arguments,too-many-function-args,too-many-public-methods
+from azure.cli.command_modules.keyvault.keyvaultclient import KeyVaultClient as _KeyVaultClient
+
+class KeyVaultClient(object):
+
+    def __init__(self, credentials, api_version='2015-06-01', accept_language='en-US', long_running_operation_retry_timeout=30, generate_client_request_id=True, filepath=None):
```

Comment:
Ultimately, this will be packaged with the data plane SDK, so it will be revved and versioned along with the SDK.